test/NuGet.Core.Tests/NuGet.Protocol.Tests/NuGet.Protocol.Tests.csproj

```diff
@@ -16,6 +16,10 @@
     <None Remove="RemoteRepositories\NewFolder\**" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Remove="compiler\resources\EntityFrameworkSearchWithStringTypes.json" />
```

Comment:
This is already done on line 9.

Reply:
Weird, it seems VS must have added this when I added a new file.